doc/SETUP.md

```diff
@@ -4,19 +4,22 @@
 
 - Install dependencies\
   `$ yarn install`
+- Create development .env file
+  `$ cp .env.example .env`
+- Set your desired .env settings to connect to either regtest or production for your specific browser.
```

Comment:
do we also have a setup to connect with a local development Rails app?

Reply:
We are going to undo this change, so you can use the same setup instructions as before (just set one env variable to point at your locally running API)